frontend/src/api/SampleData.js

```diff
@@ -3,56 +3,270 @@ import OutChannelInfo from "../model/OutputChannelInfo";
 import InChannelInfo from "../model/InputChannelInfo";
 
 export function getSenders(callback) {
-    
-    var sampleOutputChannels = [
-        new OutChannelInfo(1,"Output ch 1", 500, null),
-        new OutChannelInfo(2,"Output ch 2", 456, null),
-        new OutChannelInfo(3,"Output ch 3", 800, null)
-    ]
-    var extras = ["Additional Device details go here"];
-    var sampleSenders = [
-        new DeviceInfo("1:10:111:999", null, "123:456", "Sender 1", "Online", sampleOutputChannels, extras),
-        new DeviceInfo("1:20:111:999", null, "123:456", "Sender 2", "Error", sampleOutputChannels, extras),
-        new DeviceInfo("1:30:111:999", null, "123:456", "Sender 3", "Offline", sampleOutputChannels, extras),
-        new DeviceInfo("1:40:111:999", null, "123:456", "Sender 4", "Pending", sampleOutputChannels, extras),
-        new DeviceInfo("1:50:111:999", null, "123:456", "Sender 5", "Online", sampleOutputChannels, extras),
-        new DeviceInfo("1:60:111:999", null, "123:456", "Sender 6", "Error", sampleOutputChannels, extras),
-        new DeviceInfo("1:70:111:999", null, "123:456", "Sender 7", "Offline", sampleOutputChannels, extras),
-        new DeviceInfo("1:80:111:999", null, "123:456", "Sender 8", "Pending", sampleOutputChannels, extras),
-        new DeviceInfo("1:90:111:999", null, "123:456", "Sender 9", "Online", sampleOutputChannels, extras)
-    ];
+  const sampleOutputChannels = [
+    new OutChannelInfo(1, "Output ch 1", 500, null),
+    new OutChannelInfo(2, "Output ch 2", 456, null),
+    new OutChannelInfo(3, "Output ch 3", 800, null),
```

Comment:
Can we remove SampleData.js from the linting/sopnarqube? It's causing duplication issues and the formatting has destroyed the legibility

Reply:
There may have been a way to refactor the creation of our sample data objects using a factory pattern to eliminate code duplication, but seeing that this code is low value sample code I gladly went ahead with your recommendation of ignoring code duplication